### PR TITLE
Creating an All Menu dropdown in menu manager

### PR DESCRIPTION
**Pull request by infograf768**
Replacing https://github.com/joomla/joomla-cms/pull/9231 which had conflicts.

@rdeutz 
ready to merge as it was set to RTC

### CODE REVIEW

File: administrator/components/com_menus/controllers/items.php [hunk 1]
```diff
@@ -157,7 +159,136 @@ public function setDefault()
...
+						)
+				);
+	}
+
+	/**
+	 * Method to publish a list of items
+	 *
+	 * @return  void
+	 *
+	 * @since   12.2
```
Comment (andrepereiradasilva) [inline administrator/components/com_menus/controllers/items.php]: 12.2?

File: administrator/components/com_menus/controllers/items.php [hunk 2]
```diff
@@ -157,7 +159,136 @@ public function setDefault()
...
+			)
+		);
+	}
+
+	/**
+	 * Check in of one or more records.
+	 *
+	 * @return  boolean  True on success
+	 *
+	 * @since   12.2
```
Comment (andrepereiradasilva) [inline administrator/components/com_menus/controllers/items.php]: 12.2?

**Comment by infograf768** [comment]
Indeed... changed to 3.6.0

**Comment by infograf768** [comment]
I know where that came from: A copy/paste from the library. :smiley:

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on f9e42b51bdaae1ab88073b280369caa5c04d57ac<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10190">issues.joomla.org/joomla-cms/10190</a>.</sub>

**Comment by brianteeman** [comment]
RTC as its just a redo<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10190">issues.joomla.org/joomla-cms/10190</a>.</sub>

**Pull request merged**